docs/references/javascript/user/user.mdx

```diff
@@ -547,16 +547,13 @@ function setProfileImage(params: SetProfileImageParams): Promise<ImageResource>
     await clerk.load()
 
     if (clerk.user) {
-      // Get the current organization
-      const currentOrganization = clerk.organization
-
       // Update the user's profile image
       document.getElementById('upload-image').addEventListener('click', () => {
         const profileImage = document.getElementById('profile-image').files[0]
         console.log(profileImage)
 
-        currentOrganization
-          .setLogo({ file: profileImage })
+        clerk.user
+          .setProfileImage({ file: profileImage })
```

Comment:
@octoper Can you also apply this change for the next tab about `<script/>` ?

Reply:
Good catch @panteliselef!

Reply:
Done!